constants/sidebarData.ts

```diff
@@ -69,3 +69,9 @@ export type SubheaderData = {
   route: string
   tabs: string[]
 }
+
+export enum CampaignStatus {
+  DRAFT = 'Draft',
+  PUBLISHED = 'Published',
+  ARCHIVED = 'Archived',
+}
```

Comment:
put this in another file instead cos this file is for sidebar stuff. maybe can name the file `campaign.ts`

Reply:
Moved